sentry/src/db/channel.rs

```diff
@@ -70,6 +70,27 @@ pub async fn insert_channel(
         .await
 }
 
+pub async fn get_channel_by_creator(
+    pool: &DbPool,
+    id: &ChannelId,
+) -> Result<Vec<Channel>, RunError<bb8_postgres::tokio_postgres::Error>> {
+    pool.run(move |connection| {
+        async move {
+            match connection
+                .prepare("SELECT id FROM channels WHERE creator = $1")
```

Comment:
There is no limit? We don't even use `ChannelId`? Should this get single or multiple channels? it says `channel` in the function so I expect it will be single.

Reply:
Removed it as it's not needed